core/config/project_settings.h

```diff
@@ -108,6 +110,10 @@ class ProjectSettings : public Object {
 
 	Error _save_custom_bnd(const String &p_file);
 
+	PackedStringArray _get_supported_features();
+	PackedStringArray _get_unsupported_features(PackedStringArray &p_project_features);
+	PackedStringArray _trim_to_supported_features(PackedStringArray &p_project_features);
```

Comment:
I think these 3 methods could be made `const`, double-check that.

Reply:
Done, I added `const` everywhere it was relevant in this PR.